projects/1.16/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -0,0 +1,496 @@
+  "item.tropicraft.ashen_mask_oblongatron": "延髓面具",
+  "item.tropicraft.ashen_mask_screw_attack": "飞镖面具",
+  "item.tropicraft.ashen_mask_square_horn": "羊角面具",
+  "item.tropicraft.ashen_mask_square_zord": "方格面具",
+  "item.tropicraft.ashen_mask_the_brain": "大脑面具",
+  "item.tropicraft.ashen_mask_the_heart": "心型面具",
+  "item.tropicraft.ashen_mask_warthog": "疣猪面具",
+  "item.tropicraft.ashen_spawn_egg": "面具投手刷怪蛋",
+  "item.tropicraft.azurite_gem": "石青宝石",
+  "item.tropicraft.bamboo_item_frame": "竹制物品展示框",
+  "item.tropicraft.bamboo_mug": "竹杯",
+  "item.tropicraft.bamboo_spear": "竹矛",
+  "item.tropicraft.bamboo_stick": "竹棍",
+  "item.tropicraft.black_beach_float": "黑色沙滩浮排",
+  "item.tropicraft.black_chair": "黑色椅子",
+  "item.tropicraft.black_coffee": "黑咖啡",
+  "item.tropicraft.black_pearl": "黑珍珠",
+  "item.tropicraft.black_umbrella": "黑色伞",
+  "item.tropicraft.blow_gun": "吹气枪",
+  "item.tropicraft.blue_beach_float": "蓝色沙滩浮排",
+  "item.tropicraft.blue_chair": "蓝色椅子",
+  "item.tropicraft.blue_umbrella": "蓝色伞",
+  "item.tropicraft.brown_basilisk_lizard_spawn_egg": "棕色蜥蜴刷怪蛋",
+  "item.tropicraft.brown_beach_float": "棕色沙滩浮排",
+  "item.tropicraft.brown_chair": "棕色椅子",
+  "item.tropicraft.brown_umbrella": "棕色伞",
+  "item.tropicraft.caipirinha": "卡琵利亚",
+  "item.tropicraft.cocktail": "鸡尾酒",
+  "item.tropicraft.coconut_chunk": "椰子片",
+  "item.tropicraft.coconut_water": "椰子汁",
+  "item.tropicraft.coffee_berry": "咖啡果",
+  "item.tropicraft.cooked_fish": "熟鱼",
+  "item.tropicraft.cooked_frog_leg": "熟青蛙腿",
+  "item.tropicraft.cooked_ray": "熟鳐鱼",
+  "item.tropicraft.cowktail_spawn_egg": "牛爱花刷怪蛋",
+  "item.tropicraft.cubera_spawn_egg": "巴西笛鲷刷怪蛋",
+  "item.tropicraft.cyan_beach_float": "青色沙滩浮排",
+  "item.tropicraft.cyan_chair": "青色椅子",
+  "item.tropicraft.cyan_umbrella": "青色伞",
+  "item.tropicraft.dagger": "匕首",
+  "item.tropicraft.dolphin_spawn_egg": "海豚刷怪蛋",
+  "item.tropicraft.eagle_ray_spawn_egg": "鹰鳐刷怪蛋",
+  "item.tropicraft.eih_spawn_egg": "复活节岛石像刷怪蛋",
+  "item.tropicraft.eudialyte_axe": "异性石斧",
+  "item.tropicraft.eudialyte_gem": "异性石宝石",
+  "item.tropicraft.eudialyte_hoe": "异性石锄",
+  "item.tropicraft.eudialyte_pickaxe": "异性石镐",
+  "item.tropicraft.eudialyte_shovel": "异性石铲",
+  "item.tropicraft.eudialyte_sword": "异性石剑",
+  "item.tropicraft.exploding_coconut": "椰子炸弹",
+  "item.tropicraft.failgull_spawn_egg": "海鸟刷怪蛋",
+  "item.tropicraft.fiddler_crab_spawn_egg": "招潮蟹刷怪蛋",
+  "item.tropicraft.fire_boots": "防火靴子",
+  "item.tropicraft.fire_chestplate": "防火胸甲",
+  "item.tropicraft.fire_helmet": "防火头盔",
+  "item.tropicraft.fire_leggings": "防火护腿",
+  "item.tropicraft.fishing_net": "渔网",
+  "item.tropicraft.fresh_marlin": "生马林鱼",
+  "item.tropicraft.frog_leg": "青蛙腿",
+  "item.tropicraft.frox_conch": "Frox海螺",
+  "item.tropicraft.grapefruit": "西柚",
+  "item.tropicraft.gray_beach_float": "灰色沙滩浮排",
+  "item.tropicraft.gray_chair": "灰色椅子",
+  "item.tropicraft.gray_umbrella": "灰色伞",
+  "item.tropicraft.green_basilisk_lizard_spawn_egg": "绿色蜥刷怪蛋",
```

```suggestion
  "item.tropicraft.green_basilisk_lizard_spawn_egg": "绿色蜥蜴刷怪蛋",
```
与上方`entity`统一
是否是某种双冠蜥（*Basiliscus* sp.）？

---

此处为保留官库中文
采用 [蛇怪蜥蜴](https://baike.baidu.com/item/%E8%9B%87%E6%80%AA%E8%9C%A5%E8%9C%B4/8044593) 译法。也有称为 [双冠蜥](https://baike.baidu.com/item/%E5%8F%8C%E5%86%A0%E8%9C%A5/1317990)，但前者拉丁文学名与游戏英文对应
```suggestion
  "item.tropicraft.green_basilisk_lizard_spawn_egg": "绿色蛇怪蜥蜴怪蛋",
```